lib/reporters/GruntCoverage.js

```diff
@@ -0,0 +1,64 @@
+define([
+    'dojo/node!istanbul/lib/collector',
+    'dojo/node!istanbul/lib/object-utils',
+    'dojo/node!grunt'
+], function (Collector, utils, grunt) {
+    function GruntCoverage(config) {
+        this.config = config || {};
+        config.threshold = config.threshold || { overall: 75 };
```

Comment:
I’d probably recommend getting defaults from `istanbul/lib/report/common/defaults.watermarks()` and use the low watermark as the default for failure, for consistency.